static/js/tools/statsvar_menu.tsx

```diff
@@ -37,10 +38,15 @@ interface NodePropType {
   cd: NodeJsonField[]; // children
   t: string; // type
   sv: string[]; // statsVar dcid
+  d?: string[]; // possible denominators
   selectedNodes: StatsVarNode; // path to all the selected statsVars
   nodePath: string[]; // path to current node
   addStatsVarTitle: (statsVarId: string, statsVarName: string) => void; // pass the title of selected statsVar to parent
-  addStatsVar: (statsVar: string, nodePath: string[]) => void; // function for adding statsVar
+  addStatsVar: (
+    statsVar: string,
+    nodePath: string[],
+    denominators: string[]
+  ) => void; // function for adding statsVar
```

Comment:
can you check if this affects other client like choropleth tool which also uses the menu

Reply:
You are right! It did break the choropleth tool. fixed now

Reply:
Not because of the extra param though. The problem was that the values of StatsVarNodes become objects